fixtures/public-path/sku.config.js

```diff
@@ -1,8 +1,9 @@
 import { makeStableHashes } from '@sku-private/test-utils';
 
 export default {
-  publicPath: '/static',
-  target: 'dist/static',
```

Comment:
I assume the intent of this config was to output static assets into `/static`, but practically this isn't how our apps are typically configured.